prometheus_client/samples.py

```diff
@@ -34,6 +34,23 @@ def __lt__(self, other: "Timestamp") -> bool:
         return self.nsec < other.nsec if self.sec == other.sec else self.sec < other.sec
 
 
+class BucketSpan(NamedTuple):
+    offset: int
+    length: int
+
+
+class NativeHistStructValue(NamedTuple):
```

Comment:
Just a thought, we could probably just call this `NativeHistogram` rather than adding `StructValue`. For initial implementation we might want to prefix with `_` as well to these new types to make it clear they are internal only and may change. Or add a comment to that effect.